Khan/perseus
# feat(LEMS-2526): update aria label to include appearance descriptions

## Summary:
Updates the aria description to include visual traits

New aria label structure will be:
`Element Type` `Visible Label(s)` `Math Details` `Visual Traits`

Note: 
- Other updates to aria descriptions will be submitted in separate PRs

Issue: LEMS-2526

## Test plan:
1. Navigate to[ storybook ](https://650db21c3f5d1b2f13c02952-kiuwtyujyg.chromatic.com/?path=/docs/perseuseditor-editorpage--docs)
2. Add interactive graph widget using the add widget dropdown
3. Scroll down to add locked figure - select any from the dropdown
4. Click auto generate button to see basic auto generated aria label (AL)
5. Update color / line style / fill
6. Re-generate AL by clicking auto generate button to see updated label text

### Expected Behavior
`Visual Traits` of each of the locked figures should change when an input (color, line style, fill) is updated and the generate aria label button is pressed

## Screen recording

https://github.com/user-attachments/assets/9fb228d9-a86e-4c3b-b288-398ced6fb6f2
